helper/impls/helper-impl-14/src/main/java/com/linkedin/avroutil1/compatibility/avro14/codec/CachedResolvingDecoder.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this class was not copied from avro so i dont think the avro license header belongs here